src/detail-page/DetailPage.tsx

```diff
@@ -7,6 +7,7 @@ import { Tabs } from './components/tabs/Tabs';
 
 import { ITProfessions } from '../data/ITProfessions';
 import { ProfessionCard } from '../shared/components/ProfessionCard';
+import { DemoChart } from '../home-page/components/DemoChart';
```

Comment:
I think detail-page folder will be more suitable for DemoChart component because it is not use in home-page :)

Reply:
Good point, I've must have missed it. It's moved.